CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [10.2.0] - 25-10-20
 
+### Added
+
+- React Native THEOplayer now supports [React Compiler](https://react.dev/learn/react-compiler).
+  - For Expo users: refer to the [Expo docs](https://docs.expo.dev/guides/react-compiler/) to set up React Compiler in your app.
+
```

Comment:
Since the actual work was already done in #659, I've updated the changelog for the (already released) version 10.2.0.